include/deal.II/base/time_stepping.templates.h

```diff
@@ -241,6 +241,232 @@ namespace TimeStepping
+            this->b.push_back(0.184896052186740);
+            this->b.push_back(0.569933660509768);
+
+            std::vector<double> tmp;
+            tmp = {{0.755726351946097, 0.386954477304099}};
+            this->a.push_back(tmp);
+            break;
+          }
+        case (LOW_STORAGE_RK_STAGE5_ORDER4):
+          {
+            this->n_stages = 5;
+            this->b        = {{1153189308089. / 22510343858157.,
+                        1772645290293. / 4653164025191.,
+                        -1672844663538. / 4480602732383.,
+                        2114624349019. / 3568978502595.,
+                        5198255086312. / 14908931495163.}};
+            std::vector<double> ai;
+            ai = {{970286171893. / 4311952581923.,
+                   6584761158862. / 12103376702013.,
+                   2251764453980. / 15575788980749.,
+                   26877169314380. / 34165994151039.}};
+            this->a.push_back(ai);
+            break;
+          }
+        case (LOW_STORAGE_RK_STAGE7_ORDER4):
+          {
+            this->n_stages = 7;
+            this->b        = {{0.0941840925477795334,
+                        0.149683694803496998,
+                        0.285204742060440058,
+                        -0.122201846148053668,
+                        0.0605151571191401122,
+                        0.345986987898399296,
+                        0.186627171718797670}};
+            std::vector<double> ai;
+            ai = {{0.241566650129646868 + this->b[0],
+                   0.0423866513027719953 + this->b[1],
+                   0.215602732678803776 + this->b[2],
+                   0.232328007537583987 + this->b[3],
+                   0.256223412574146438 + this->b[4],
+                   0.0978694102142697230 + this->b[5]}};
+            this->a.push_back(ai);
+            break;
+          }
+        case (LOW_STORAGE_RK_STAGE9_ORDER5):
+          {
+            this->n_stages = 9;
+            this->b        = {{2274579626619. / 23610510767302.,
+                        693987741272. / 12394497460941.,
+                        -347131529483. / 15096185902911.,
+                        1144057200723. / 32081666971178.,
+                        1562491064753. / 11797114684756.,
+                        13113619727965. / 44346030145118.,
+                        393957816125. / 7825732611452.,
+                        720647959663. / 6565743875477.,
+                        3559252274877. / 14424734981077.}};
+            std::vector<double> ai;
+            ai = {{1107026461565. / 5417078080134.,
+                   38141181049399. / 41724347789894.,
+                   493273079041. / 11940823631197.,
+                   1851571280403. / 6147804934346.,
+                   11782306865191. / 62590030070788.,
+                   9452544825720. / 13648368537481.,
+                   4435885630781. / 26285702406235.,
+                   2357909744247. / 11371140753790.}};
+            this->a.push_back(ai);
```

Comment:
is a always a single column? If yes, you can change it to just store ai directly or not?

Reply:
Yes, it is always a single column and the vector is from a base class, so I didn't change it.